react/components/ShelfItem.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedNumber } from 'react-intl'
+
+export default class ShelfItem extends Component {
+  static propTypes = {
+    imageUrl: PropTypes.string,
+    name: PropTypes.string,
+    price: PropTypes.number,
+    productLink: PropTypes.string,
+  }
+
+  static contextTypes = {
+    culture: PropTypes.object,
+  }
+
+  render() {
+    const { imageUrl, name, price, productLink } = this.props
+    return (
+      <div className="w-20">
+        <a
```

Comment:
This should be a `Link` component from `render` with `page` and `params` properties to allow client side navigation :) e.g.: https://github.com/vtex-apps/render-getting-started/blob/master/react/index.js#L43